orchestra/tests/test_query_params_filter.py

```diff
@@ -0,0 +1,93 @@
+from rest_framework.test import APIRequestFactory
+from rest_framework.viewsets import ModelViewSet
+
+from orchestra.models import Todo
+from orchestra.models import TodoQA
+from orchestra.tests.helpers.fixtures import TodoFactory
+from orchestra.tests.helpers.fixtures import ProjectFactory
+from orchestra.orchestra_api import build_url_params
+from orchestra.todos.filters import QueryParamsFilterBackend
+from orchestra.todos.serializers import TodoQASerializer
+from orchestra.todos.serializers import BulkTodoSerializer
+from orchestra.tests.helpers import OrchestraTransactionTestCase
+
+
+factory = APIRequestFactory()
+
+
+class DummyView(ModelViewSet):
+    model = TodoQA
+    serializer_class = TodoQASerializer
+    queryset = TodoQA.objects.all()
+
+
+class DummyView2(ModelViewSet):
+    model = Todo
+    serializer_class = BulkTodoSerializer
+    queryset = Todo.objects.all()
+    filterset_fields = ('project__id',)
+
+
+class QueryParamsFilterBackendTests(OrchestraTransactionTestCase):
+    def _get_qs_kwargs(self, view_class, url_params):
+        url = '/some-path/{}'.format(url_params)
+        view = view_class(action_map={'get': 'list'})
+        request = factory.get(url)
+        request = view.initialize_request(request)
+        view.request = request
+        view.format_kwarg = {}
+        backend = QueryParamsFilterBackend()
+        params = request.query_params.dict()
+        qs_kwargs = backend._get_filter_kwargs(view, params)
+        filterset_fields_kwargs = backend._get_filterset_fields_kwargs(
+            view, params, qs_kwargs)
+        return qs_kwargs, filterset_fields_kwargs
+
+    def test_random_key_values(self):
+        url_params = build_url_params(
+            123, 'slug', **{'some_random_key': 'some_value'})
+        qs_kwargs, _ = self._get_qs_kwargs(DummyView, url_params)
```

Comment:
Is there a way to test this on `GenericTodoViewset` instead on a `DummyView`? It would be good if these tests fail if a changes allows non-serializer fields to be passed?